tee-worker/ts-tests/integration-tests/common/utils/context.ts

```diff
@@ -27,7 +27,10 @@ export async function initWorkerConnection(endpoint: string): Promise<WebSocketA
     return wsp;
 }
 
-export async function initIntegrationTestContext(substrateEndpoint: string): Promise<IntegrationTestContext> {
+export async function initIntegrationTestContext(
+    substrateEndpoint: string,
```

Comment:
can you rename `substrateEndpoint` to `parachainEndpoint`? Should be more clear 🙌